git_workflow.md

```diff
@@ -0,0 +1,166 @@
+# Git workflow
```

Comment:
Is there a reason this is duplicated?  It is generally a bad idea to bifurcate files, because which is the one we should be maintaining?

I suggest to delete this file.